neo/EndPointConfig.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Net;
+
+namespace Neo
+{
+    public class EndPointConfig
```

Comment:
Use `IPEndPoint` directly?

Reply:
was my first thought, but maybe it could be harder when we parse the json file, but i can do it